cves/CVE-2018-8014.yml

```diff
@@ -103,9 +114,13 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: |
+    There were unit tests for the CORS filter although they were not testing for
```

Comment:
Im not sure if this was covered before but what does CORS stand for?

Reply:
CORS stands for Cross-Origin Resource Sharing.

Reply:
Can you explain why it is needed in this subsystem?

Reply:
The CORS Filter is a implementation of W3C's CORS specification which is what enables cross-origin requests, this is used the Catalina subsystem as it is an implementation of Java Servlet.